plugins/ja3_fingerprint/ja3_utils.cc

```diff
@@ -0,0 +1,102 @@
+/** @ja3_utils.cc
+  Plugin JA3 Fingerprint calculates JA3 signatures for incoming SSL traffic.
+  @section license License
```

Comment:
Probably good to space this out a bit, like here:
https://github.com/apache/trafficserver/blob/master/src/proxy/http/HttpSM.cc#L3